src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/__tests__/helpers.ts

```diff
@@ -46,15 +46,27 @@ export const fields: TestField[] = [
   ...fieldTypes.map((type) => ({
     name: `${camelCase(type)}Field`,
     type,
+    userDefined: false as false,
```

Comment:
Why do we need the as here?

Reply:
Without it TS creates an error because it interprets it as a Boolean whereas ESQLFieldWithMetadata specifically requires `false`. I think `as const` would work too if preferred